MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift

```diff
@@ -54,7 +54,7 @@ class ReminderNotifications {
 
     static let titleString = String(
         format: NSLocalizedString(
-            "You've been sent %@!",
+            "scheduled-reminder-title %@",
```

Comment:
You should parameters inside the string resource, not here. The location of the parameters could be different for different languages.

Reply:
it's just stub to make it clear that there is a variable.

Reply:
I see, saw the convo about this after I made this comment. @Jasonvdb what do you think?

Reply:
`scheduled_reminder.title`

Reply:
It does look a little weird having that in the key. But I understand what @SerhiiShovkoplias means here. Your call, I don't feel too strongly either way

Reply:
I see the point too, I agree, I don't feel strongly either. If it'll be explicit having a suffix like `.with_param` might read better. `scheduled_reminder.title.with_param`.